src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -1043,7 +1138,26 @@ public static void addToZipFile(final String fileName, final ZipOutputStream zos
             /**
              * Intentionally not being thrown.
              */
-            Log.getLogger().warn("Error packing " + fileName + " into the zip.");
+            Log.getLogger().warn("Error packing " + file.getName() + " into the zip.");
+        }
+    }
+
+    /**
+     * Removes oldest backup files to reduce kept number as desired
+     */
+    private static void pruneOldBackups(final int numToKeep)
+    {
+        if (numToKeep > 0)
+        {
+            final File[] backupFiles = getBackupSaveDirectory().listFiles();
+            if (backupFiles != null && backupFiles.length > numToKeep)
+            {
+                Arrays.sort(backupFiles);
+                for (int i = 0; i < backupFiles.length - numToKeep; ++i)
+                {
+                    backupFiles[i].delete();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Do something with the "boolean" value returned by "delete". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS899)